shared/src/db/database.ts

```diff
@@ -485,7 +487,9 @@ class database extends Dexie {
      * @returns
      */
     fromIsoDateTime(date: string) {
-        return DateTime.fromISO(date).toMillis();
+        return DateTime.fromISO(date)
+            .setLocale(navigator.language || "en-US")
+            .toMillis();
```

Comment:
not sure if the locale needs to be set when converting Luxon dates to Unix dates, but guess it does not do damage either.